cpp/rss_aggregator.cc

```diff
@@ -191,7 +193,7 @@ void PerformDescriptionSubstitutions(const std::string &patterns_and_replacement
 
 
 // \return the number of new items.
-unsigned ProcessFeed(const std::string &feed_id, const std::string &feed_name, const std::string &feed_url,
+int ProcessFeed(const std::string &feed_id, const std::string &feed_name, const std::string &feed_url,
```

Comment:
If the return type is changed, the comment above this line should be changed as well.